tools/image_processing/imagej2/imagej2_convert_format/imagej2_convert_format.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="imagej2_convert_format" name="Convert image format" version="1.0.0">
+    <description>with bioformats</description>
+    <macros>
+        <import>imagej2_macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command interpreter="python">
+        imagej2_convert_format.py
+        -i $input
```

Comment:
The `$input` and `$output` needs to be quoted, the parameters doesn't need to. More or less all strings, paths needs to be quoted.